api/handler/contact.go

```diff
@@ -27,6 +27,16 @@ func contact(router chi.Router) {
 	})
 }
 
+// nolint: gofmt,goimports
+//
+//	@summary	Gets all Moira contacts
+//	@id			get-all-contacts
+//	@tags		contact
+//	@produce	json
+//	@success	200	{object}	dto.ContactList					"Contacts fetched successfully"
```

Comment:
Tabs look broken. It's okay or should we raplcae them with spaces?

Reply:
They are crookedly displayed in PR, if you do a pull of my branch you'll see that the tabs are normal there, that's why `swag fmt` is called after generating the documentation